mdoc/Test/en.expected-vbnet/FrameworksIndex/One.xml

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Framework Name="One">
+  <package Id="thepackage" Version="2.2.2" />
```

Comment:
Not sure if I'm missing something, but I see that you modified three of the `expected` integration tests to include `<package Id="thepackage" Version="2.2.2" />`, but I don't see where this value is coming from in a `frameworks.xml` file ... not sure if/what I'm missing, or if my gaze just passed over the file in question.

@asokolovskiy15 ... can you help me understand which file this value is coming from, in the tests?

Reply:
@joelmartinez I investigated this issue and concluded, that this file is not under version control system. So, I fixed it and added a new integration test.